db/migrate/20170405065305_remove_image_password_audit.rb

```diff
@@ -0,0 +1,20 @@
+class RemoveImagePasswordAudit < ActiveRecord::Migration
+  def up
+    Audited::Audit.where(:auditable_type => 'Image').find_in_batches do |batch|
+      batch.each do |audit|
+        audit.audited_changes.delete("password")
```

Comment:
no point deleting if we destroy the audit (in case it's just the password that's audited) anyways I think?